spatialos-sdk/src/worker/op.rs

```diff
@@ -462,17 +502,17 @@ pub struct EntityQueryResponseOp {
 }
 
 #[derive(Debug)]
-pub struct AddComponentOp {
+pub struct AddComponentOp<'a> {
     pub entity_id: EntityId,
     pub component_id: ComponentId,
-    component_data: component::internal::ComponentData,
+    component_data: component::internal::ComponentData<'a>,
 }
 
-impl AddComponentOp {
+impl<'a> AddComponentOp<'a> {
     pub fn get<C: Component>(&self) -> Option<&C> {
         if C::ID == self.component_data.component_id {
             // TODO: Deserialize schema_type if user_handle is null.
-            Some(unsafe { &*(self.component_data.user_handle as *const _) })
+            Some(unsafe { &*(self.component_data.user_handle as *const _ as *const _) })
```

Comment:
Is the double cast necessary?

Reply:
Nope! It was briefly necessary in a previous iteration, but I changed things such that it's no longer necessary. Looks like I missed one when I was cleaning it up.